t/sanity.t

```diff
@@ -468,3 +468,59 @@ lock 2: unlock: nil, unlocked
 --- no_error_log
 [error]
 
+
```

Comment:
I think we should add test cases for

1. a newly created object that never locks any key or 
2. an object it has unlocked its key previously held.